src/net/sourceforge/kolmafia/Expression.java

```diff
@@ -24,6 +24,12 @@
 public class Expression {
   private static final Pattern NUM_PATTERN = Pattern.compile("([+-]?[\\d.]+)(.*)");
   private static final int STACK_SIZE = 128;
+  private static final char[] binaryOpcodes;
+
+  static {
+    binaryOpcodes = new char[] {'+', '-', '*', '/', '%', '^', '<', '≤', '>', '≥', 'x'};
```

Comment:
`combine()` is operating on the bytecodes rather than the expression string. The "input expression" that generates them are `gte(expr1, expr2)` and `lte(expr1, expr2)`

They don't look to be used in any of the data files though!

Interestingly though, when checking, I've also missed `m` (min) as one of the valid binary opcodes

Reply:
I added them within the last year. We're running real short on opcodes!